spec/linter-xmllint-spec.coffee

```diff
@@ -114,6 +114,19 @@ describe 'The xmllint provider for Linter', ->
           expect(messages.length).toEqual 1
           expect(messages[0].range).toEqual [[3, 2], [3, 21]]
     waitsForPromise ->
+      return atom.workspace.open(__dirname + '/fixtures/invalid/xml-model-unavailable.xml').then (editor) ->
+        return lint(editor).then (messages) ->
+          expect(messages.length).toBeGreaterThan 0
+          expect(messages.length).toBeLessThan 3
```

Comment:
@Arcanemagus Another question regarding Coffeescript: is there a better way to express this - I don't like the fact that it mentions `0` and `3` while the actual interval is `[1, 2]`? I would imagine something like `.toBeIn [1, 2]` or at least `.toBeGreaterOrEqualThen 1` and `.toBeLessOrEqualThan 2` but couldn't find anything.